src/components/user/resolvers.js

```diff
@@ -4,8 +4,22 @@ const Task = require("../task/model");
 
 module.exports = {
   Query: {
-    user: async (parent, args, context, info) => {
-      return "user query";
+    getUser: async (parent, args, { authUser, db: { User } }, info) => {
+      if (!authUser) {
+        throw new CustomGraphQLerror(StatusCodes.UNAUTHORIZED);
+      }
+
+      const user = await User.findById(authUser.userId);
```

Comment:
You can use destructuring assignment here and take advantage of it in the return statement.

Such like return {name, settings, lastLoginDate}

Reply:
Agreed. Updated.